app/assets/stylesheets/standards/_classes.sass

```diff
@@ -12,3 +12,28 @@
 
 .bg-pink
   background-color: $yellow-color
+
+.bg-light-gray
+  background-color: $light-gray
+
+.bg-medium-gray
+  background-color: $medium-gray
+
+.bg-dark-gray
+  background-color: $dark-gray
+
+.mb-1rem
+  margin-bottom: 1rem
+
+.ml-onehalf
+  margin-left: 1.5rem
+
+.all-caps
+  text-transform: uppercase
+
+.border-right-white
+  border-right: solid $white 1px
+
+.img-responsive
```

Comment:
Instead of using a class for this, we can style the `img` tag with `max-width`, like that:
```
img
  max-width: 100%
```

That, together with the `meta name="viewport" content="initial-scale=1.0, width=device-width"` that we already have on our layout, will make all images responsive.